app/src/androidTest/java/ch/epfl/sweng/hostme/AddFragmentTest.java

```diff
@@ -61,6 +62,10 @@ public static void setUp() {
         FirebaseApp.initializeApp(ApplicationProvider.getApplicationContext());
     }
 
+    @Rule
+    public GrantPermissionRule accessRule = GrantPermissionRule.grant(android.Manifest.permission.CAMERA, Manifest.permission.WRITE_EXTERNAL_STORAGE, Manifest.permission.INTERNET, Manifest.permission.READ_EXTERNAL_STORAGE);
```

Comment:
using full path to `CAMERA` permission when Manifest is imported

Reply:
I tried to add permission that are working on AccountFragment tests but it doesn't seem to work...